content/docs/command-reference/remote/add.md

```diff
@@ -140,9 +140,19 @@ $ dvc remote modify myremote endpointurl \
                     https://object-storage.example.com
 ```
 
-> See `dvc remote modify` for a full list of S3 API parameters.
+To save and reuse key and secret for all your dvc project, you could add them to
+the global dvc config file:
 
-S3 remotes can also be configured entirely via environment variables:
+```dvc
+$ dvc remote modify --global myremote access_key_id <my-access-key>
+$ dvc remote modify --global myremote secret_access_key <my-secret-key>
```

Comment:
I'm a bit lost where the original source of these two keywords, but they are what work instead of `aws_access_key_id`/`aws_secret_access_key`

Reply:
this is general thing for any S3 ... the recommended way to achieve what you are trying to do (the way I understand this) is to setup AWS cli util or manually create `~/.aws/credentials`. `boto` (a lib we use internally will pick it up) automatically. Rarely you wound need to override keys/secrets via DVC config this way.

Reply:
> I'm a bit lost where the original source of these two keywords, but they are what work instead of

they come from the `remote modify` docs - https://dvc.org/doc/command-reference/remote/modify (S3 section)

Reply:
> but they are what work instead of `aws_access_key_id`/`aws_secret_access_key`

@efiop it's mighta  good point to actually use prefixes like `aws_` - we can name them the same way as they are named for a specific provider then. To be specific - if provider has prefixes we should probably use the same name. It's indeed convenient to have aws_access_key_id, AWS_ACCESS_KEY_ID, and the same name in DVC config.